File: hackathon-backend/src/main/java/hackathon/entity/Project.java
package hackathon.entity;

import javax.persistence.*;
import javax.validation.constraints.NotNull;

@Entity
@Table(name="project")
public class Project {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id = 0L;

    @NotNull
    @Column(name = "project_code", nullable = false)
    private String projectCode;

    @NotNull
    @Column(name = "client_input", nullable = false)
    private String clientInput;

    @NotNull
    @Column(name = "team_size", nullable = false)
    private Long teamSize;

    public String getProjectCode() {
        return projectCode;
    }

    public void setProjectCode(String projectCode) {
        this.projectCode = projectCode;
    }

    public String getClientInput() {
        return clientInput;
    }

    public void setClientInput(String clientInput) {
        this.clientInput = clientInput;
    }

    public Long getTeamSize() {
        return teamSize;
    }

    public void setTeamSize(Long teamSize) {
        this.teamSize = teamSize;
    }

  ;
}

File: hackathon-backend-kotlin/src/main/kotlin/hackathon/backend/kotlin/entity/Project.kt
package hackathon.backend.kotlin.entity

import javax.persistence.*
import javax.validation.constraints.NotNull

@Entity
@Table(name = "project")
data class Project (
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long = 0L,

    @NotNull
    @Column(name = "project_code", nullable = false)
    var projectCode :String,

    @NotNull
    @Column(name = "client_input", nullable = false)
    var clientInput:  String,

    @NotNull
    @Column(name = "team_size", nullable = false)
    var teamSize:  Long
)
